src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/events/PullRequestDeclinedWebhookEvent.java

```diff
@@ -0,0 +1,27 @@
+package com.atlassian.bitbucket.jenkins.internal.trigger.events;
+
+import com.atlassian.bitbucket.jenkins.internal.model.BitbucketPullRequest;
+import com.atlassian.bitbucket.jenkins.internal.model.BitbucketUser;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.annotation.Nullable;
+import java.util.Date;
+
+/**
+ * @since 3.0.0
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PullRequestDeclinedWebhookEvent extends PullRequestClosedWebhookEvent {
```

Comment:
I'm not strongly against this change- my main opposition was the work required to do it, but I am curious about what value it has. When does our plugin ever need to distinguish between a `closed` and a `declined` event?

Reply:
I think there is weak reasons for it
1. to be consistent, in that there is an event for each state
2. the API only offers interactions based on specific API states (or specific IDs)
3. This in extension allows us to have only one eventId per event, not have multiples which simplifies the code immensely in other places